README.md

```diff
@@ -25,14 +25,26 @@ Manticore is a symbolic execution tool for analysis of binaries and smart contra
 
 Manticore can analyze the following types of programs:
 
-- Linux ELF binaries (x86, x86_64 and ARMv7)
 - Ethereum smart contracts (EVM bytecode)
+- Linux ELF binaries (x86, x86_64 and ARMv7)
+
+Manticore includes a symbolic Ethereum Virtual Machine (EVM) and a convenient interface for automated compilation and analysis of Solidity. It integrates with [Ethersplay](https://github.com/trailofbits/ethersplay), Trail of Bits’ visual disassembler for EVM bytecode, for analysis visualization.
```

Comment:
I feel like this is a little too much detail for this part of the readme, but i'm not sure where it would go. maybe try to incorporate the content into the below sections somehow?

"Use the CLI to conveniently explore possible states in Solidity smart contracts using Manticore's symbolic Ethereum Virtual Machine." <- that would take care of the first sentence. not sure where we would put the second one about ethersplay (or if it really even belongs here). i feel like it is a better fit for a wiki page or something, and not the readme.